zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -132,24 +141,26 @@ object Server {
     keepAlive: Boolean = true,
     consolidateFlush: Boolean = false,
     flowControl: Boolean = true,
+    httpContentDecompression: (Boolean, Boolean) = null,
```

Comment:
```suggestion
    httpContentDecompression: (Boolean, Boolean) = null,
```
Set default values here.